docs/concepts/stateful-stream-processing.md

```diff
@@ -242,6 +246,42 @@ updates to that state.
 See [Restart Strategies]({% link dev/task_failure_recovery.md
 %}#restart-strategies) for more information.
 
+### Unaligned Checkpointing
```

Comment:
Should we mention that this is an experimental feature?
I think it should be a separate statement in the end of the section.

Reply:
My understanding was that this document is rather an expanded glossary and talks about the concept and not the implementation. Thus, I'd leave the implementation state out of this place. The ops link will directly say that it's experimental in 1.11.

Reply:
The page already goes quite deep into the details so I don't see why it shouldn't be mentioned here. Some users could benefit by ruling out the feature earlier if they are considering Flink or its configuration.

Reply:
Since we decided at a different point to drop the experimental label, I'd leave this section as is.